staking/availability/interface.go

```diff
@@ -0,0 +1,28 @@
+package availability
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	staking "github.com/harmony-one/harmony/staking/types"
+)
+
+// Reader ..
+type Reader interface {
+	ReadValidatorSnapshot(
+		addr common.Address,
+	) (*staking.ValidatorWrapper, error)
+}
+
+// Header is the interface of block.Header for calculating the BallotResult.
+type Header interface {
```

Comment:
`RoundReader`

Reply:
Thanks for naming recommendation. Fixed.